src/style.css

```diff
@@ -1,3 +1,70 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable color-hex-case */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+a {
+  display: block;
+}
+
+.logo__link {
+  height: 40px;
+}
+
+.header-container {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.item__link {
+  padding: 22.5px 0;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.item__link:hover {
+  color: #00ACDC;
+}
+
+.item__link:active {
```

Comment:
The :active state usually has a different color or effect than the hover state to provide better UX. Consider changing the color or adding another effect.